src/chrome/content/rules/Stack-Exchange.xml

```diff
@@ -138,10 +140,10 @@
 	<securecookie host="^teststackoverflow\.com$" name=".+" />
 
 <!-- Rules -->
-	
-	<!-- https links from other pages to these will cause MCB for important elements, hence the downgrades -->
-	<rule from="^https://([\w.-]+)\.([\w-]+)\.stackexchange\.com/" 
-		to="http://$1.$2.stackexchange.com/" downgrade="1" />
+
+	<!-- meta.* sites moved to *.meta - we can safely redirect to their new equivalents, which support https -->
+	<!-- details: https://meta.stackexchange.com/questions/292058/network-wide-https-its-time -->
+	<rule from="^https://meta\.([\w-]+)\.stackexchange\.com/" to="https://$1.meta.stackexchange.com/" />
 
 	<rule from="^http://www\.stackoverflow\.blog/" to="https://stackoverflow.blog/" />
```

Comment:
The redirect from `meta.*` to `*.meta` doesn't happen in HTTPS though, in fact it can't because they don't have valid certificates for `meta.*`. So I'm okay with rewriting it here.